pkg/apis/trust/v1alpha1/types_bundle.go

```diff
@@ -118,10 +118,31 @@ type BundleTarget struct {
 type AdditionalFormats struct {
 	// JKS requests a JKS-formatted binary trust bundle to be written to the target.
 	// The bundle is created with the hardcoded password "changeit".
```

Comment:
suggestion: Looks like this comment will need updating. Might be worth linking to https://cert-manager.io/docs/faq/#keystore-passwords as well!